pkg/controllers/dashboard/chart/chart.go

```diff
@@ -40,20 +51,48 @@ type Definition struct {
 	RemoveNamespace     bool
 }
 
-// RancherConfigGetter is used to get Rancher chart configuration information from the rancher config map
+// RancherConfigGetter is used to get Rancher chart configuration information from the rancher config map.
 type RancherConfigGetter struct {
 	ConfigCache corev1.ConfigMapCache
 }
 
 // GetPriorityClassName attempts to retrieve the priority class for rancher pods and feature charts as set via helm values.
 func (r *RancherConfigGetter) GetPriorityClassName() (string, error) {
-	configMap, err := r.ConfigCache.Get(namespace.System, settings.ConfigMapName.Get())
+	return r.getKey(PriorityClassKey, settings.ConfigMapName.Get())
+}
+
+// GetWebhookValues attempts to retrieve the desired webhook Values from the rancher-config configMap.
+func (r *RancherConfigGetter) GetWebhookValues() (map[string]any, error) {
```

Comment:
Nit: Might be good to replace this with a generic "GetChartValues(chartName string)" method that we can use with the webhook's chart name, so that this can be re-used easier by other areas.